src/slices/BondSlice.ts

```diff
@@ -190,22 +190,33 @@ export const bondAsset = createAsyncThunk(
 
     // Deposit the bond
     let bondTx;
+    let uaData = {
+      address: address,
+      value: value,
+      type: "Bond",
+      bondName: bond,
```

Comment:
this should probably be `bond.displayName`, right?

Reply:
correct! pushing a change.